File: pangram/pangram.ts
/**
 * Проверяет, является ли строка панграммой
 * @param sentence Входная строка
 * @returns true если строка содержит все буквы алфавита, иначе false
 */

function isPangram(sentence: string): boolean {
    const alphabet = 'abcdefghijklmnopqrstuvwxyz';
    const lowerSentence = sentence.toLowerCase();
    return alphabet.split('').every(letter => lowerSentence.includes(letter));
}

console.log(isPangram("thequickbrownfoxjumpsoverthelazydog")); // true
console.log(isPangram("whatdoesthefoxsay")); // false

File: pangram/pangram.swift
func isPangram(_ sentence: String) -> Bool {
    /**
     * Проверяет, является ли строка панграммой
     * @param sentence Входная строка
     * @return true если строка содержит все буквы алфавита, иначе false
     */
     
    let lowerSentence = sentence.lowercased()
    for c in "abcdefghijklmnopqrstuvwxyz" {
        if !lowerSentence.contains(c) {
            return false
        }
    }
    return true
}

print(isPangram("thequickbrownfoxjumpsoverthelazydog")) // true
print(isPangram("whatdoesthefoxsay")) // false